kaggle_cli/download.py

```diff
@@ -46,13 +48,35 @@ def download_file(self, browser, url):
         self.app.stdout.write('downloading %s\n' % url)
         local_filename = url.split('/')[-1]
         headers = {}
+        file_size = 0
+        total_size = 0
+        bar = progressbar.ProgressBar()
 
         if os.path.isfile(local_filename):
             file_size = os.path.getsize(local_filename)
             headers['Range'] = 'bytes={}-'.format(file_size)
 
+        with closing(browser.get(url, stream=True)) as stream_orig:
```

Comment:
using `head` method is better https://github.com/floydwch/kaggle-cli/blob/master/kaggle_cli/download.py#L53-L55 .